operations/app/src/modules/network/main.tf

```diff
@@ -1,12 +1,12 @@
 /* Subnet CIDRs */
 module "subnet_addresses" {
-  for_each = data.azurerm_virtual_network.vnet
+  for_each = toset(local.vnet_names)
```

Comment:
This was changed as in higher environments Terraform was running into a chicken and egg problem with the plan. By using the local variable, Terraform is able to anticipate the resources it needs to create before the `apply`.